### PR TITLE
update rank subline with past week difference

### PR DESCRIPTION
**Pull request by greg-schrammel**
Fixes BX-####
Figma link (if any):

## Screen recordings / screenshots

Screenshot 2024-01-30 at 15 06 50

### CODE REVIEW

File: src/entries/popup/pages/home/Points/usePoints.ts [hunk 1]
```diff
@@ -25,7 +28,7 @@ export const fetchPointsQuery = async (address: Address) =>
 let nextDropTimeout: NodeJS.Timeout | undefined;
 export const usePoints = (address: Address) => {
   const query = useQuery({
-    queryKey: ['points', address],
+    queryKey: ['points', address, PointsDocument.loc?.source.body],
```
Comment (greg-schrammel) [inline src/entries/popup/pages/home/Points/usePoints.ts:31]: updating the queryKey when the query changes dismisses the cache and force a refetch, so we don't have to check if `last_airdrop` exists every time

**Comment by DanielSinclair** [comment]
@greg-schrammel Seeing a direction inversion here; this wallet went down 24 this week I believe. 
Screenshot 2024-01-30 at 3 40 43 PM

**Review by DanielSinclair — APPROVED** [review]
Looks good to me!

File: src/entries/popup/pages/home/Points/PointsDashboard.tsx [hunk 2]
```diff
@@ -495,10 +524,9 @@ export function PointsDashboard() {
   const { currentAddress } = useCurrentAddressStore();
   const { data: points } = usePoints(currentAddress);
 
-  const hasLastAirdropPoints =
-    points?.user.stats.last_airdrop?.differences.some(
-      (d) => d && d.earnings.total > 0,
-    );
+  const hasLastAirdropPoints = points?.user.stats.last_airdrop.differences.some(
```
Comment (BrodyHughes) [inline src/entries/popup/pages/home/Points/PointsDashboard.tsx:527]: jw, why remove the `?` on the `last_airdrop`?

File: src/entries/popup/pages/home/Points/usePoints.ts (hunk 1 above)
Comment (BrodyHughes) [reply src/entries/popup/pages/home/Points/usePoints.ts:31]: oh i see

**Review by BrodyHughes — APPROVED** [review]
i don't think this is related here, but do you see this issue? 
> New wallet 
> signed up for points 
> points page doesn't update
> sign up again
> says 'existing user'
> points page still doesn't update

<video src="https://github.com/rainbow-me/browser-extension/assets/41711440/2654d5d8-d6a8-4adf-9ac7-c4c58de9fdbf" />

**Pull request merged**